.github/workflows/kcp-image.yaml

```diff
@@ -25,13 +25,18 @@ jobs:
         id: vars
         run: echo "::set-output name=sha_short::$(echo ${{ github.sha }} | cut -b -7)"
 
+      - name: Extract branch name
+        shell: bash
+        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
```

Comment:
We should probably be consistent in our set-output commands? i.e. `::` instead of `##`?

Reply:
I just saw that ${{ github.ref_name }} should now provide the branch name.
Regarding consistency I was wondering why kcp-image,yaml uses buildah  and syncer-image.yaml uses ko.

Reply:
> Regarding consistency I was wondering why kcp-image,yaml uses buildah and syncer-image.yaml uses ko.

Let's use a separate issue for that, please. Mind filing one for a discussion?

Reply:
> Let's use a separate issue for that, please. Mind filing one for a discussion?
For the records, here is [the link](https://github.com/kcp-dev/kcp/issues/902) to the issue